nxpprog.py

```diff
@@ -397,10 +399,19 @@
 
 prog = None
 
+class LogType(IntEnum):
+    INFO = 0
+    DEBUG = 1
+    ERROR = 2
 
-def log(str):
-    sys.stderr.write("%s\n" % str)
-    sys.stderr.flush()
+
+def log(str, type):
+    if type == LogType.INFO:
+        logging.info(str)
```

Comment:
Any reason why we don't just use the `logging.info` calls directly vs passing a string and type?

Reply:
Removed log function and enum types, logging directly now